src/components/MapGuide/index.tsx

```diff
@@ -1,101 +1,74 @@
-import { useEffect, useRef, useState } from 'react';
 import Button from '@mui/material/Button';
-import DialogContent from '@mui/material/DialogContent';
-import DialogContentText from '@mui/material/DialogContentText';
-import Paper, { PaperProps } from '@mui/material/Paper';
-import Backdrop, { BackdropProps } from '@mui/material/Backdrop';
-import Draggable from 'react-draggable';
-import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
 
 import {
-    StyledMapGuideDialog,
     StyledMapGuideButton,
-    StyledDialogTitle,
-    CloseButton,
+    StyledTooltipTitle,
+    StyledTooltipContainer,
 } from './styled';
 
-const PaperComponent = (props: PaperProps) => {
+import React from 'react';
```

Comment:
Starting from React v17 you no longer have to import React unless you need to use hooks or other components / functions from `react` library